XRTK-Core/ProjectSettings/ProjectSettings.asset

```diff
@@ -398,7 +398,7 @@ PlayerSettings:
     m_APIs: 0b00000011000000
     m_Automatic: 1
   - m_BuildTarget: WindowsStandaloneSupport
-    m_APIs: 1100000002000000
+    m_APIs: 0200000011000000
```

Comment:
Why this platform change?

Reply:
Because the editor was opening in the incorrect graphics mode (OpenGL instead of DirectX)